### PR TITLE
Feat: new UI add page params

### PR DESCRIPTION
**Pull request by ga-devfront**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Add hardcoded required parameters for new UI on controller and updating twig according new parameters.
| Type?             | new feature
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | N/A
| Sponsor company   | @PrestaShopCorp
| How to test?      | For dev : you can change parameters on controller and reload according page to see changes. For test : you can just check if all new ui pages work fine.

### CODE REVIEW

**Review by M0rgan01 — DISMISSED** [review]

**Review by Quetzacoalt91 — DISMISSED** [review]

**Review by Quetzacoalt91 — DISMISSED** [review]

**Review by tblivet — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
